python/ray/dashboard/subprocesses/routes.py

```diff
@@ -240,3 +106,60 @@ async def parent_side_handler(
             return handler
 
         return _wrapper
+
+
+def decorate_stream_handler(handler):
```

Comment:
Do we need the wrappers? I feel now each handler can just returns the proper aiohttp response.

Reply:
Currently the handlers for streaming response and websocket response are async generators yielding `str` or `bytes`. If we don't use wrapper, then they need to return aiohttp response directly.

Reply:
Yea I think they can just return aiohttp response directly. It returns async generator because the old implementation needs to send them to MP.queue but we now don't need that.